.github/workflows/package.yml

```diff
@@ -63,3 +81,20 @@ jobs:
         CONAN_VISUAL_VERSIONS: ${{ matrix.vs-version }}
         CONAN_APPLE_CLANG_VERSIONS: "11.0"
         CONAN_GCC_VERSIONS: ${{ matrix.gcc-version }}
+      if: startsWith(matrix.python-arch, 'arm') != true
+    - name: Generating conan user directory and building the solution
+      run: |
+        conan user
+        python build.py
+      env:
+        CONAN_REFERENCE: "ZenGitHub/1.0"
+        CONAN_USERNAME: "jonico"
+        CONAN_LOGIN_USERNAME: "jonico"
+        CONAN_CHANNEL: "stable"
+        CONAN_UPLOAD: "https://api.bintray.com/conan/conan-jonico/libzengithub"
+        CONAN_STABLE_BRANCH_PATTERN: "release/*"
+        CONAN_VISUAL_VERSIONS: ${{ matrix.vs-version }}
+        CONAN_APPLE_CLANG_VERSIONS: "11.0"
+        CONAN_GCC_VERSIONS: ${{ matrix.gcc-version }}
+        CONAN_ARCHS: ${{ matrix.python-arch }}
+      if: startsWith(matrix.python-arch, 'arm')
```

Comment:
This looks like a lot of code duplication. Would the other environments break if you added the `CONAN_ARCHS`env for them?

Reply:
it would break, since conan arch are not python arch, e.g. conan autodetect come up with both x86 and x86_64 for x86_64.

for arm it looks like it fails to detect and tries to build for x86 :(
https://github.com/spreequalle/libzengithub/runs/531279338?check_suite_focus=true

less code duplication, here
https://github.com/spreequalle/libzengithub/blob/master/.github/workflows/package.yml#L70

Reply:
not sure about the upload though

Reply:
> less code duplication, here
> https://github.com/spreequalle/libzengithub/blob/master/.github/workflows/package.yml#L70

That looks good, would accept this in a FUP PR

- [x]  Address in-line feedback regarding code duplication in the workflow file

> not sure about the upload though

[Looking into the build log](https://github.com/spreequalle/libzengithub/runs/517876201#step:7:283) it appears as if the version of libzengithub referred to has already been cached:

> ZenGitHub/1.0@jonico/stable:d286bc363f1ec1891f6eb391dbadf6cee665c357 - Cache

Could it be that you did not clean up your runner between builds and hence a previous upload attempt of what you tried to build failed, conan is not trying to do so again or is that checksum associated to a conan package I already uploaded for Linux?